lib/OpenQA/WebAPI.pm

```diff
@@ -129,6 +129,8 @@ sub startup {
     $apik_auth->post('/')->to('api_key#create');
     $apik_auth->delete('/:apikeyid')->name('api_key')->to('api_key#destroy');
 
+    $r->get('/search')->name('search')->to('search#search');
```

Comment:
This is the user-facing webpage for the search, comprised of 1) the slug 2) the route name 3) the module/sub (controller and implementation of the route respectively)

Reply:
If I understand correctly, there should be a module such as `search.pm` in controller and a `sub search`, but I do not find this in your pr.

Reply:
It's `Search.pm` in the `API/V1` namespace.

Reply:
I think I mixed up my explanation there because I had the other route at the back of my head (the `search#query` one on the API). I will be more explicit:

- There is `templates/webapi/search/search.html.ep` which is maps to `search#search`.
- There's `lib/OpenQA/WebAPI/Controller/API/V1/Search.pm` which has the `query` method.